codespell_lib/data/dictionary.txt

```diff
@@ -22387,8 +22387,8 @@ excentuating->accentuating
 exceopt->exempt
 exceopted->exempted
 exceopts->exempts
-exceotion->exemption
-exceotions->exemptions
```

Comment:
Originates in #1429.

I agree, and I suspect this is the result of an error within the fix. I would suggest:
1. putting `exception` before `exemption`,
2. fixing all verbal forms of `exceopt` too.

Reply:
Yes, please do.

Reply:
I think the initial intent was with a **p** after **o**::
```
exceoption->exemption
exceoptions->exemptions
```
to match:
```
exceopt->exempt
exceopted->exempted
exceopts->exempts
```

Reply:
So I would recommend fixing all of:
* `exceopt` (all verbal forms, suggest `exempt` first and `except` second?)
* `exceoption` (also plural, suggest `exemption` first and `exception` second?)
* `exceot` (all verbal forms, suggest `except` first and `exempt` second?)
* `exceoption` (also plural, suggest `exception` first and `exemption` second?)

Reply:
I agree with the suggestions, and force-pushed a commit aligning all of these.